components/list/list-item-button-mixin.js

```diff
@@ -43,6 +43,22 @@ export const ListItemButtonMixin = superclass => class extends ListItemMixin(sup
 				text-align: start;
 				width: 100%;
 			}
+			/** clean up with flag GAUD-7495-list-interactive-content */
+			:host(:not([_list-item-interactive-enabled])) button {
+				background-color: transparent;
+				border: none;
+				cursor: pointer;
+				display: block;
+				height: 100%;
+				outline: none;
+				width: 100%;
+			}
+			/** clean up with flag GAUD-7495-list-interactive-content */
+			:host(:not([_list-item-interactive-enabled]):not([button-disabled]):not([no-primary-action])) [slot="content"],
+			:host(:not([_list-item-interactive-enabled]):not([no-primary-action])) [slot="control-action"] ~ [slot="content"],
+			:host(:not([_list-item-interactive-enabled]):not([no-primary-action])) [slot="outside-control-action"] ~ [slot="content"] {
+				pointer-events: none;
+			}
```

Comment:
copy/pasted back in from [here](https://github.com/BrightspaceUI/core/pull/5520/files#diff-690b268628d93c7861b05c76da7607613913aebaa0129b3c0d455dd33407e33c) then checked for the flag's property.

I kept the flag on and off cases separate so enable easy clean up.